packages/slice-machine/pages/slices.tsx

```diff
@@ -45,8 +45,8 @@ const CreateSliceButton = ({
 
 const SlicesIndex = ({ env }: { env: Environment }) => {
   const libraries = useContext(LibrariesContext);
-  const [data, setData] = useState({ loading: false });
-  const [isOpen, setIsOpen] = useState(false);
+  const [isCreatingSlice, setCreatingSlice] = useState<boolean>(false);
```

Comment:
I have for habbit to have the same name between the set and the value.
`setIsCreatingSlice` not mandatory at all but i just noticed it